pomegranate/gmm.pyx

```diff
@@ -167,17 +170,20 @@ cdef class GeneralMixtureModel(BayesModel):
 			else:
 				improvement = log_probability_sum - last_log_probability_sum
 
+				time_spent = time.time() - epoch_start_time
 				if verbose:
-					print("Improvement: {}".format(improvement))
+					print("Improvement: {} in {:.2f}s".format(improvement, time_spent))
```

Comment:
@jmschrei I am not sure if you meant it this way in your comment (I believe the `s` at the end was omitted), but the reason why I tried to split this into two lines was that currently the line is a bit too long when you set your `tabwidth` to 8.

However, since there are even longer lines in this file, this is acceptable in my mind too.